cub/benchmarks/nvbench_helper/nvbench_helper.cu

```diff
@@ -561,7 +561,7 @@ thrust::device_vector<T> gen_uniform_offsets(seed_t seed,
   segment_offsets[total_elements] = total_elements + 1;
   thrust::exclusive_scan(segment_offsets.begin(), segment_offsets.end(), segment_offsets.begin());
   typename thrust::device_vector<T>::iterator iter =
-    thrust::find_if(segment_offsets.begin(), segment_offsets.end(), gt_t<T>{total_elements});
+    thrust::find_if(segment_offsets.begin(), segment_offsets.end(), ge_t<T>{total_elements});
```

Comment:
I think the logic for this change is for the following scenario. Assume we're generating offsets for a total of `100` items. If the offset sequence is:
`..., 95, 100, 105`, we want to make the second last offset (i.e., `100`) to be the last offset in the offset array. Previously, we would have generated `..., 95, 100, 100`.

Reply:
Just to make sure, does benchmark data generator affects cub tests?

Reply:
No, I just came across this because I had bluntly copied the implementation for the test-side offset generation. So, I thought I'd also fix it in the original implementation.

Reply:
At some point, it'd be nice to hoist the data generator to CCCL level, so we could use it in CUB / Thrust tests and benchmarks.